android-services/src/main/kotlin/drivechain/androidservices/AndroidSystemServiceModule.kt

```diff
@@ -62,67 +62,67 @@ import javax.inject.Singleton
  * Use this to inject android's system services directly into your app instead of injecting a whole
  * context directly.
  */
-@Module class AndroidSystemServiceModule(private val context: Context) {
```

Comment:
So this is a breaking change, which is a real bummer.

Is there any way we can provide functionality for both the new android injections and the old module system?